6_visualize/src/create_gif_makefile.R

```diff
@@ -10,7 +10,7 @@ create_intro_gif_makefile <- function(makefile, task_plan, remake_file) {
     file_extensions=c('feather','ind'))
 }
 
-create_storm_gif_makefile <- function(makefile, task_plan, remake_file) {
+create_storm_gif_makefile <- function(makefile, task_plan, remake_file, final_targets) {
```

Comment:
The `final_targets`argument doesn't appear to be used in this call to `create_task_makefile`, but that's where it should appear (along with a value for `finalize_funs`) if any combining is going to happen...do you think this may be a case where we started to try to add a combiner and then aborted partway through?

Reply:
If I'm remembering right, these code changes may have been the point at which you created this scipiper issue https://github.com/USGS-R/scipiper/issues/116.

Reply:
You are right, that is exactly what happened. I got part way and then had to figure out a different approach. This relates to our pipeline overhaul plan which is something we want to do soon, but not an immediate priority. I will delete this so that there isn't confusion.